package.json

```
@@ -5,14 +5,17 @@
   "dependencies": {
     "@material-ui/core": "^4.6.0",
     "@material-ui/icons": "^4.5.1",
+    "@reduxjs/toolkit": "^1.0.4",
```

```suggestion
    "@reduxjs/toolkit": "1.0.4",
```
much prefer version pinning. at best I am only comfortable with ~ instead of ^